CHANGELOG.md

```diff
@@ -5,6 +5,10 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased]
+- GeneralDataStructure transporter enhanced
+  - Deep ndarray to list serialization
+  - Deep list to ndarray deserialization 
+- LabelBinerizer transporter updated
```

Comment:
Follow the pattern used in other projects (see [pycm's changelog](https://raw.githubusercontent.com/sepandhaghighi/pycm/master/CHANGELOG.md)). Something like below would be appropriate:
```
### Added
- `ndarray_to_list` method added to `GeneralDataStructureTransporter`
- `list_to_ndarray` method added to `GeneralDataStructureTransporter` 
### Changed
- GeneralDataStructure transporter enhanced
- LabelBinerizer transporter updated
```

Reply:
Sure, applied.